.github/workflows/label-checker.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     - run: exit 0
       name: 'Monojenkins PR'
       # always happy if monojenkins
-      if: github.actor == 'monojenkins'
+      if: github.actor == 'monojenkins' || github.actor == 'vs-mobiletools-engineering-service2'
```

Comment:
could not `github.actor == 'monojenkins'` be removed ?

Reply:
@spouliot could, but should I? I did think about it replacing it, I have no strong feeling

Reply:
Landing as it, but created issue: https://github.com/xamarin/xamarin-macios/issues/10359